doc/quick-start.rst

```diff
@@ -255,6 +255,42 @@ Write this in your ``jbuild`` file:
 
 And run the tests with:
 
+.. code:: bash
+
+    jbuilder runtest
+
+Build and run multiple targets
+==============================
```

Comment:
In this section, it's not very clear to me what is meant by building and running targets. What the jbuild file really does is define two executables and run them as part of the test suite.